packages/cli/src/controller/init-controller.ts

```diff
@@ -2,24 +2,68 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import fs from 'fs';
+import * as path from 'path';
+import {promisify} from 'util';
+import yaml from 'js-yaml';
 import rimraf from 'rimraf';
-import simpleGit, {SimpleGit} from 'simple-git';
-const git: SimpleGit = simpleGit();
+import simpleGit from 'simple-git';
 
-const starterPath = 'https://github.com/OnFinality-io/subql-starter';
+const STARTER_PATH = 'https://github.com/OnFinality-io/subql-starter';
 
-export async function createProject(projectName: string): Promise<void> {
-  const localPath = `${process.cwd()}/${projectName}`;
-  if (fs.existsSync(localPath)) {
-    throw new Error('Directory exists');
+export interface projectSpec {
```

Comment:
Capital case for interfaces, classes

Reply:
better move this definition to `src/types.ts` since it is project-wise required.